nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -104,6 +105,11 @@ controller_server:
       xy_goal_tolerance: 0.25
       yaw_goal_tolerance: 0.25
       stateful: True
+    precise_goal_checker:
```

Comment:
I'm not sure if we should make these changes to the main branch (adding another goal checker that's not used).

Reply:
I agree, this was just to show the idea how two different goal checkers could be changed.

Reply:
I think that's valuable in a tutorial or docs once we get this up and running